modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -0,0 +1,80 @@
+= Download and Install Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+image::install-service-mesh-breadcrumb.png[]
+
+Before you install Service Mesh, you must first download the product from the installer download URL. As part of installation, the following required components are deployed on a Kubernetes cluster:
+
+* Istio
+* Service Catalog
+* Mulesoft Anypoint Service Mesh Broker
+* Mulesoft ClusterIP Service
+ 
+ 
+The following topics are covered in this section:
+
+* Download Service Mesh Client
+* Install Service Mesh
+
+== Download Anypoint Service Mesh Client
+
+You install, manage, and troubleshoot your Anypoint Service Mesh using Service Mesh Client. 
+
+To download the Service Mesh Client, perform the following step:
+ 
+Type the following command at the command prompt:
+ 
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > service-mesh && chmod +x service-mesh`
+
+The Service Mesh Client is now downloaded on your local drive.
+
+== Install Anypoint Service Mesh
+
+When you being the installer, Istio is first installed, and the MuleSoft adapter and broker are then installed inside the Istio mixer. The installer perform several steps as part of the installation, such as it checks for dependencies and various configurations.
+
+To install Anypoint Service Mesh, perform the following steps:
+
+. From the location where you downloaded your Service Mesh Client, type ./service-mesh install
+The Welcome to Service Mesh installer message is displayed.
+. At the Please enter your Client Id prompt, specify your client ID that obtained as part of the prerequisites.
+. At the Please enter your Client Secret prompt, specify your client secret that obtained as part of the prerequisites.
+. At the Please enter the path to your Service Mesh license, specify the MuleSoft license that you obtained by emailing the MuleSoft team as part of the prerequisites.
++
+*Installation Command Example:*
```

Comment:
the command is incorrect? clientId and clientSecret shouldn't have spaces right?